neural_lam/vis.py

```diff
@@ -64,11 +66,18 @@ def plot_error_map(errors, data_config, title=None, step_length=3):
 
 @matplotlib.rc_context(utils.fractional_plot_bundle(1))
 def plot_prediction(
-    pred, target, obs_mask, data_config, title=None, vrange=None
+    pred,
+    target,
+    obs_mask,
```

Comment:
Unused argument. No need to include in each call anymore.

Reply:
Done in https://github.com/leifdenby/neural-lam/commit/b41ed2f98b551f26fe0d7f8c4e1fb00181130b7d